src/jv_unicode.c

```diff
@@ -118,3 +118,16 @@ int jvp_utf8_encode(int codepoint, char* out) {
   assert(out - start == jvp_utf8_encode_length(codepoint));
   return out - start;
 }
+
+// space codepoints for unicode basic latin and latin-1 supplement
```

Comment:
This is same as what golang `strings.TrimSpace`/`unicode.IsSpace` considers whitespace https://pkg.go.dev/unicode#IsSpace

Reply:
Note that Go's `strings.TrimSpace` considers `White_Space` property as well.